### DISCUSSION TITLE
MGEN Compile Issue

### DISCUSSION THREAD

**Discussion by mushbi1234** [USNavalResearchLaboratory/mgen#50] [General]
I am having an issue compiling MGEN on Ubuntu. I installed libcap-dev but I am still getting /usr/bin/ld: cannot find -lpcap: No such file or directory. What do I need to do to fix this error?

Thanks
Denny

<img width="725" alt="Screenshot 2023-06-22 at 2 29 51 PM" src="https://github.com/USNavalResearchLaboratory/mgen/assets/92055614/737d1a36-995c-4197-b44b-32d04d49b658">

**Comment by ljt-git** [comment]
Did you install the wrong library perhaps?  You need libpcap-dev vs libcap-dev.  

 

From: mushbi1234 ***@***.***>
Reply-To: USNavalResearchLaboratory/mgen ***@***.***>
Date: Thursday, June 22, 2023 at 5:34 PM
To: USNavalResearchLaboratory/mgen ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [USNavalResearchLaboratory/mgen] MGEN Compile Issue (Discussion #50)

 

I am having an issue compiling MGEN on Ubuntu. I installed libcap-dev but I am still getting /usr/bin/ld: cannot find -lpcap: No such file or directory. What do I need to do to fix this error?

Thanks
Denny

**Comment by mushbi1234** [comment]
Thanks. That fixed it.
